app/views/plans/_edit_details.html.erb

```diff
@@ -60,6 +64,7 @@
           <%= f.label(:identifier, _('ID'), class: 'control-label') %>
         </div>
         <div class="col-md-8">
+          <em class="sr-only"><%= _('A pertinent ID as determined by the funder and/or organisation.') %></em>
```

Comment:
It would be better to re-use a variable here too

Reply:
@xsrust missed that one. Will change and push.

Reply:
@xsrust Update code with two changes requested.

Reply:
Thanks @johnpinto1 
I wondered if you'd done those first